### ISSUE TITLE
External file $refs not supported by jump to definition

### ISSUE THREAD

**Issue by dodongo** [42Crunch/vscode-openapi#13]
A ref like the following doesn't support jump to definition, though it seems like it should be supported by the [spec](https://swagger.io/docs/specification/using-ref/):

```
"schema": {
  "$ref": "./common_schemas.json#/CustomType"
}
```

where `common_schemas.json` is in the same directory and contains:

```
{
  "CustomType": {
    "description": "A custom type.",
    "type": "string"
  }
}
```

**Comment by ak1394** [comment]
You are correct, current implementation does not support opening external files. I'll look into supporting this in the next release.

**Comment by dodongo** [comment]
@ak1394 One other note. Not sure if this would end up getting covered by the same fix, but in terms of getting the overview OpenAPI pane reflecting all the currently used data types - i.e: 

<img width="302" alt="Screen Shot 2019-08-24 at 1 12 58 PM" src="https://user-images.githubusercontent.com/557625/63640667-f2950500-c670-11e9-81e4-784f335ba69c.png">

to reflect components that are in external refs, would the standard practice be to have the component still be listed in a specific openapi.json spec, and then referenced externally? i.e.:

```
"components": {
    "schemas": {
      "CustomType": {
        "$ref": "./common_schemas.json#/CustomType"
      },
...
```

I would image the tooling would have no other way of actually displaying what components are being used on the overview pane. Unless there's plans for a separate section for external types?

**Comment by ak1394** [comment]
Indeed. For the time being the outline is pretty simple and it pretty much follows the structure of the OpenAPI file itself. Operations for example don't list their parameters in the outline at all and components/parameters simply display parameter names without making distinction between inline/external definitions. This is something for us to figure out for the future versions.

**Comment by ak1394** [comment]
The $refs issue has been fixed in the new v1.8.10. I'm going to close this issue, but feel free to comment (or open new issue) if you have further thoughts on displaying of types in the outline.
